# HG changeset patch
# User Rik <rik@octave.org>
# Date 1467816918 25200
#      Wed Jul 06 07:55:18 2016 -0700
# Branch stable
# Node ID de18e547e99d0eb437a6ef66a66e9f19aa7b61df
# Parent  837d7e48c9b4a154dc2e89831ce723219ee92d14
Link Z_LIB to liboctinterp (bug #48407).

* libinterp/link-deps.mk: Add $Z_LIBS to link dependencies for liboctinterp.

diff --git a/libinterp/link-deps.mk b/libinterp/link-deps.mk
--- a/libinterp/link-deps.mk
+++ b/libinterp/link-deps.mk
@@ -5,16 +5,17 @@ if AMCOND_ENABLE_DYNAMIC_LINKING
 else
   LIBOCTINTERP_LINK_DEPS = $(DLDFCN_LIBS)
 endif
 
 LIBOCTINTERP_LINK_DEPS += \
   $(FT2_LIBS) \
   $(HDF5_LIBS) \
   $(MAGICK_LIBS) \
+  $(Z_LIBS) \
   $(FFTW_XLIBS) \
   $(OPENGL_LIBS) \
   $(X11_LIBS) \
   $(CARBON_LIBS) \
   $(GL2PS_LIBS) \
   $(LLVM_LIBS) \
   $(JAVA_LIBS) \
   $(LAPACK_LIBS)
